kofta/src/app/modules/room-chat/EmoteData.ts

```diff
@@ -319,12 +319,6 @@ export const customEmojis = [
     keywords: ["crypto", "doge", "dodgycoin"],
     imageUrl: "/emotes/dodgyCoin.png",
   },
-  {
-    name: "blaze",
-    short_names: ["blaze"],
-    keywords: ["blaze", "yes", "snark"],
-    imageUrl: "/emotes/blaze.png",
-  },
```

Comment:
This shouldn't be removed.

Reply:
Must have pulled an outdated branch, will fix

Reply:
All good. 👍🏼